zeton/api/points.py

```diff
@@ -22,7 +22,7 @@
 @bp.route("/child/<child_id>/points/add/<points>/<ex_id>", methods=['POST'])
 @auth.login_required
 @auth.logged_child_or_caregiver_only
-def add_points(child_id,points,ex_id):
+def add_points_buttons(child_id,points,ex_id):
```

Comment:
Wydaje mi że można to zorbić na jednym edpoincie. Proponuję zostawić `add_points`. Poprawić jedynie ścieżkę w formtce np:
`                <h2>Dodawaj punkty</h2>
                <form action="/api/child/{{ child.id }}/points/add/0/0" method="post">
                    <div class="form-group button">
                        Liczba punktów: <input class="form-control" type="number" name="points" required>
                        <input class="btn btn-info" type="submit" name="dodaj punkty" value="Dodaj punkty"/>
                    </div>
                </form>`

Reply:
zmienione